rollups/evm/connect-oracle/README.md

```diff
@@ -0,0 +1,66 @@
+## Foundry
```

Comment:
Readme should be more about this contract/guide than Foundry itself. Can provide forge setup in prerequisites tho